apps/transport/lib/mix/tasks/transport/import_communes.ex

```diff
@@ -69,40 +105,51 @@ defmodule Mix.Tasks.Transport.ImportCommunes do
     %{geom | srid: 4326}
   end
 
-  defp load_etalab_communes(region_insees) do
-    @communes_url
-    |> HTTPoison.get!(timeout: 15_000, recv_timeout: 15_000)
-    |> Map.fetch!(:body)
-    |> Jason.decode!()
-    |> Enum.filter(&(&1["type"] == "commune-actuelle" and &1["region"] in region_insees))
+  defp readable_changeset(changeset) do
+    changes = changeset.changes
+    data = changeset.data
+
```

Comment:
```suggestion
  defp readable_changeset(%Ecto.Changeset{changes: changes, data: data} = changeset) do
```

Je trouve que c'est mieux ainsi car :
- tu mets en évidence la structure utilisée
- moins de lignes